monitoring/route_origin_monitor.c

```diff
@@ -9,17 +9,58 @@
 /* starting point */
 uint64_t route_origin_monitor(args_t *args UNUSED);
 
+PROOF_SEAHORN_INSTS (
+
 #define ORIGIN_ATTR 1
 
 #define ORIGIN_ATTR_IGP 0
 #define ORIGIN_ATTR_EGP 1
 #define ORIGIN_ATTR_UNK 2
 
+char int2char(int c)
```

Comment:
Why don't you write this function like this ?
```c
char int2char(int c) {
    static const char *nbs = "0123456789";
    assert(c >= 0 && c <= 9);
    return nbs[c];
}
```
IMHO, it is easier to read.

Is this because of verification tools you wrote it with a succession of case statements ?